src/coupled/ESM1.5/casa_landuse.F90

```diff
@@ -230,15 +227,11 @@ SUBROUTINE newlitter_thin( casabiome,frac_x,ifpre_x,frac_y,ifpre_y, &
          dplitter(nv,metb) = dpplant(nv,leaf) + dpplant(nv,froot)-dplitter(nv,str)
          dplitter(nv,CWD) = dpplant(nv,wood)
       ENDIF  !of "icycle >2"
-    ENDIF
-  END DO
```

Comment:
I presume that you are removing these and using. the expanded loop? Fair enough, recalculating *litter_g is probably a good thing. BUT the IF condition you are inheriting is different from the original. Is this what you want?

Reply:
Sounds reasonable. So, I'll wait for that final update?

Reply:
I'm pretty sure this `IF` condition is the one that we want. There are 0's in the cover fractions where a give tile does not exist, and you were right that the minimum cover fraction doesn't cause much problem because the relevant carbon pools are 0.

Also, I rewrote the whole subroutine so that it doesn't re-scale with the cover fractions because this is pointless for wood thinning. I also applied the same conditions to the changes in the plant pools in `casa_um_init`, because this was causing a miss-match in the change in the plant pools and the change in the litter pools. This required putting another loop in for each of the PFTs there.

This might need another review from you. I have added some carbon conservation checks in the code and everything passes those at runtime. I also checked the carbon conservation on the model output and it all looks good there too.